CHANGELOG.md

```diff
@@ -28,6 +28,10 @@ The documentation should give you some information on how the various components
 - Removal of the `webnative.initialise`, `app` and `permissionedApp` functions. These have been replaced by the `program` function mentioned above.
 - Removal of the `fs.appPath` function, replaced with the `appData` function located in the path module.
 
+**Fixes**
+
+- Fixes `Could not locate user DID in DNS` uncaught exception reported after successfully linking a device
```

Comment:
Was this a problem before 0.35?

Reply:
We did not notice it before `0.35`, but I didn't see anything in the refactor that would have caused it to occur. Maybe drop it from the changelog though since it didn't manifest before?

Reply:
Yeah I’d just drop it.